### PR TITLE
Give components correct priorities

### LINKED ISSUE #541: Give components correct priorities
## User stories
As a user I the components to be called in the correct order in order to make sure that when calling a component nothing from other components is missing.

currently planned order:
Updater -> FualtInjector -> RouteController -> Rest

## Acceptance Criteria
- [ ] give every component the correct priority

### PR DESCRIPTION
**Pull request by instant-sky**
Fixes #541 

Description (what might a PO or reviewer want to know?)

## PR checklist

- [ ] Acceptance criteria fulfilled
- [ ] Additional features are tested
- [ ] Docs (code, wiki & diagrams) updated
- [ ] Breaking changes anounced
- [ ] Added classes that inherit from `BaseModel` to `src.constants.tables`
- [ ] Dev-branch has been merged into local branch to resolve conflicts
- [ ] Tests and linter have passed AFTER local merge
- [ ] Another dev reviewed and approved

### CODE REVIEW

**Review by antonykamp — COMMENTED** [review]
Thanks for the change 👍 
I have one question and one idea 😇 What priority does the SimulationObjectUpdater have, and what do you think about an Enum for priorities instead of magic numbers 🙈

**Comment by instant-sky** [comment]
The `SimulationObjectUpdatingComponent` has priority 10. It was already like this so I did not have to add it by myself. 

An enum could be an option. Would you name the priorities something like `VERY_HIGH`, `HIGH`, `MEDIUM`, `LOW` or maybe something like `FIRST`, `SECOND`, `THIRD`, ...? 🤔

**Comment by antonykamp** [comment]
> An enum could be an option. Would you name the priorities something like VERY_HIGH, HIGH, MEDIUM, LOW or maybe something like FIRST, SECOND, THIRD, ...? 🤔

I like your first proposal 😇

File: src/component.py [hunk 1]
```diff
@@ -9,14 +9,21 @@ class Component(ABC):
     event_bus: EventBus
     priority: int
 
-    def __init__(self, event_bus: EventBus, priority: int):
+    PRIORITY_LEVELS: dict[str, int] = {
```
Comment (antonykamp) [inline src/component.py:12]: I thought about real enums, actually 😇 We could compare them and use them as types. But if you are happy with this solution, then I am too.

**Review by antonykamp — APPROVED** [review]

File: src/component.py (hunk 1 above)
Comment (instant-sky) [reply src/component.py:12]: I think it is sufficient for now. But we can change that in the future if it is necessary? 🤔

**Pull request merged**